src/iterativesolvers.jl

```diff
@@ -18,7 +18,7 @@ function get_vecs!((phi, q), M, V, AV, ni)
   return lambda
 end
 
-function orthogonalize!(q::ITensor, V, ni)
+function orthogonalize!(q::ITensor, V, ni::Integer)
```

Comment:
Is the reasoning behind this change because we are now overloading the `KrylovKit` version?

Reply:
Yes. We could discuss other ways to go but without this change, in the context of overloading `KrylovKit.orthogonalize!`, the existing code was leading to an ambiguity error.

Reply:
I'll change it back if I can successfully get rid of the ITensors-level KrylovKit imports. Better to keep these changes minimal.

Reply:
This should be very cleaned up now by the latest set of changes. See comment below.